nat-lab/tests/conftest.py

```diff
@@ -363,3 +364,15 @@ def copy_file_from_container(container_name, src_path, dst_path):
         )
     except subprocess.CalledProcessError:
         print(f"Error copying log file {src_path} from {container_name} to {dst_path}")
+
+
+async def collect_mac_diagnostic_reports():
+    print("Collect mac diagnostic reports")
+    async with mac_vm_util.new_connection(reenable_nat=False) as connection:
```

Comment:
Why this `reenable_nat` is really needed? Maybe it has to be set to `False` in some other places too.

Reply:
To be honest I don't know, and I would prefer to keep the behaviour changes limited to what is needed for collecting mac diagnostic reports.

Reply:
And what happens if we set `reenable_nat=True` here?

Reply:
I don't need to disable/enable nat so even if it would work for me to do it, I don't want to do it here.

Reply:
That is true in most places, that's why I was asking, maybe we don't need to do that in other places :smile: anyway, this was more of a discussion thread, not a request change other places.